From 1ca4345aca5b70c0a4322d07076684e3205a6cdb Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Tue, 10 Jul 2018 11:53:30 +0300
Subject: [PATCH] help: fix the help message for dali-<board>-tbot

The help message states that the Linux test will be executed
for the target, while the default test case is tc_ub_help.py.
Fix the help message to reflect the actual defaults.

Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/Makefile b/Makefile
index 27a0be1..5cefee7 100644
--- a/Makefile
+++ b/Makefile
@@ -26,7 +26,7 @@
 	@echo
 	@echo 'Testing:'
 	@echo '  dali-<board>-test    - Run uboot test on <board> e.g. onix, pldm, asic'
-	@echo '  dali-<board>-tbot    - Run linux test on <board> e.g. onix, pldm, asic'
+	@echo '  dali-<board>-tbot    - Run tbot test on <board> e.g. onix, pldm, asic'
 	@echo '                           set HOST_TBOT_TC=<test name> to'
 	@echo '                             override dafault test tc_ub_help.py'
 	@echo '                           set HOST_TBOT_IP=<PC lab IP> to'
